### ISSUE TITLE
Anything special when handling r2 as a metric?

### ISSUE THREAD

**Issue by ajing** [automl/auto-sklearn#315]
I got the following warning when I set metric = autosklearn.metrics.r2

Is there anything I need to set when the metrics is r2?

`[WARNING] [2017-06-23 00:10:44,316:AutoMLSMBO(1)::8ddbf0f4e5553c9f30a62574dbf79b98] Could not find meta-data directory .../anaconda3/envs/ml_automl/lib/python3.6/site-packages/autosklearn/metalearning/files/r2_regression_dense`

Then I got a lot of Running Time Warning:
```

/anaconda3/envs/ml_automl/lib/python3.6/site-packages/autosklearn/smbo.py:737: RuntimeWarning: invalid value encountered in true_divide
  (1. - dataset_minimum))
/anaconda3/envs/ml_automl/lib/python3.6/site-packages/scipy/stats/_distn_infrastructure.py:879: RuntimeWarning: invalid value encountered in greater
  return (self.a < x) & (x < self.b)
/anaconda3/envs/ml_automl/lib/python3.6/site-packages/scipy/stats/_distn_infrastructure.py:879: RuntimeWarning: invalid value encountered in less
  return (self.a < x) & (x < self.b)
```

**Comment by mfeurer** [comment]
Your first issue is a duplicate of #280, there is simply no regression meta-data available. Auto-sklearn will run just fine, but cannot make use of any meta-learning. I also opened an upstream issue about the warning. They aren't harmful and you can simply ignore them.
